webpack.config.babel.js

```diff
@@ -35,7 +35,10 @@ const basePlugins = [
   new webpack.NoErrorsPlugin(),
   new webpack.optimize.DedupePlugin(),
   new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify(ENV)
+    'process.env': {
+      'NODE_ENV': JSON.stringify(ENV),
+      'ONFIDO_URL': JSON.stringify(process.env.ONFIDO_URL || 'https://apidev.onfido.com')
```

Comment:
I think the reason we passed the URL as a prop was because that was the only way to propagate the value to core. Therefore we could now use Devise for both.

I think basing it on NODE_ENV is dangerous in case a customer is building the SDK in a development environment. Perhaps we could have an ONFIDO_ENV?

Unrelated I think you should namespace the Onfido API  interactions and the Onfido SDK server so we have consistent names for each throughout.